test-cases/longevity/longevity-alternator-3h-multidc.yaml

```diff
@@ -0,0 +1,57 @@
+test_duration: 200
+prepare_write_cmd:
+  - >-
+    bin/ycsb load dynamodb -P workloads/workloadc -threads 80 -p recordcount=6990500
+    -p fieldcount=10 -p fieldlength=512 -p dataintegrity=true
+    -p insertstart=0 -p insertcount=3495250
+
+  - >-
+    bin/ycsb load dynamodb -P workloads/workloadc -threads 80 -p recordcount=6990500
+    -p fieldcount=10 -p fieldlength=512 -p dataintegrity=true
+    -p insertstart=3495250 -p insertcount=3495250
+
+  - >-
+    bin/ycsb load dynamodb -P workloads/workloadc -threads 80 -p recordcount=6990500
+    -p fieldcount=10 -p fieldlength=512 -p dataintegrity=true
+    -p insertstart=0 -p insertcount=3495250 -p table=usertable_no_lwt
+
+  - >-
+    bin/ycsb load dynamodb -P workloads/workloadc -threads 80 -p recordcount=6990500
+    -p fieldcount=10 -p fieldlength=512 -p dataintegrity=true
+    -p insertstart=3495250 -p insertcount=3495250 -p table=usertable_no_lwt
+
+  - cassandra-stress write cl=QUORUM n=6990500 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=80 -pop seq=1..6990500 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5
+
+stress_cmd: [
+  "cassandra-stress write cl=QUORUM duration=180m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=20 -pop seq=1..6990500 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
```

Comment:
shouldn't this be LOCAL_QUORUM ?

Reply:
right

Reply:
we will need to check this when implementing region awareness for C-S